cmd/cl-adm/templates/config.go

```diff
@@ -63,9 +73,16 @@ func (c Config) TemplateArgs() (map[string]interface{}, error) {
 		return nil, err
 	}
 
+	dataplaneTypeEnvoy := true
+
+	if c.DataplaneType == DataplaneTypeGo {
+		dataplaneTypeEnvoy = false
+	}
+
 	return map[string]interface{}{
-		"peer":       c.Peer,
-		"dataplanes": c.Dataplanes,
+		"peer":               c.Peer,
+		"dataplanes":         c.Dataplanes,
+		"dataplaneTypeEnvoy": dataplaneTypeEnvoy,
```

Comment:
See my suggestion in Slack on passing `dataplaneType` in addition to all `dataplaneType*` strings.